utils/generate-bazel-build-rules.js

```diff
@@ -100,15 +101,16 @@ const getDepLabels = (root, depMap, dependencies = {}) => {
   return Object.keys(dependencies)
     .map(name => {
       const {dir, meta} = depMap[name] || {};
-      if (dir) {
+      const version = dependencies[name];
+      if (
+        dir != null &&
+        (version.includes('*') || semver.satisfies(meta.version, version))
```

Comment:
I guess there's one edge-case where one could use `resolutions` field? That may easily affect how the dependency get resolved, so whatever is specified in `dependencies[name]` may not hold. 

_* I am not 100% sure if we should consider this a legit use-case though, as it seems that a proper way would be to set the desired dependency version in the package.json for local workspaces, and maybe only use the resolutions field to enforce it over how transitive dependencies are resolved_

Reply:
Yeah, I think technically it's an edge case, but we don't need to think about it because resolutions shouldn't be used for top level deps.